docs/source/advanced/boosted_frame.rst

```diff
@@ -239,32 +238,21 @@ and using the above formula we can calculate the interaction time in the
 boosted frame. The number of required iterations are then given by:
 
 .. math::
-  N_{boost} = \frac{T_{interact}'}{\gamma (1+\beta) \Delta t}
+  N_{boost} = \frac{T_{interact}'}{\Delta t'}
 
-using :math:`\Delta t' = \gamma (1+\beta) \Delta t` for
+where :math:`\Delta t' = \gamma (1+\beta) \Delta t` is
 the timestep in the boosted frame.
 
-As explained in detail in the next section, FBPIC defines the simulation box to move
-with the speed of light (although the moving window velocity can be slower)
-in the direction of the laser, thus the length of the simulation box, as well
-as the spatial resolution (and therefore the timestep) scales
-:math:`\propto \gamma(1+\beta)`.
-
-Of course, the above formula can be used to immediately estimate the expected
-speedup of the boosted frame simulation. In practice, however, the theoretical
-speedup can be different. As the number of particles are reduced in the boosted frame,
-the computational execution time of a single timestep can sometimes be reduced,
-resulting in an even higher speedup. In contrast, the online diagnostics
-(see :ref:`boosted_frame_lab_diagnostics`) will decrease the expected speedup
-in practice.
-
```

Comment:
While the above information is important, it is not essential at this point, in order to run boosted-frame simulations. In order to simplify the documentation and make it easier for a user to read through, I removed it.

Reply:
:white_check_mark: